src/app/(app)/home/EntryListItem.jsx

```diff
@@ -1,28 +1,27 @@
 'use client';
 import styles from './home.module.scss';
+import Link from 'next/link';
 
 export default function EntryListItem({ entry }) {
   return (
     <div className={styles.entryDayWrapper}>
       <div className={styles.dateBox}>
-        <p className={styles.dateBoxNumber}>{entry.creationDateTime.toLocaleString('default', { day: 'numeric' })}</p>
-        <p className={styles.dateBoxWeekday}>
-          {entry.creationDateTime.toLocaleString('default', { weekday: 'short' })}
-        </p>
+        <p className={styles.dateBoxNumber}>{entry.created_at.toLocaleString('default', { day: 'numeric' })}</p>
+        <p className={styles.dateBoxWeekday}>{entry.created_at.toLocaleString('default', { weekday: 'short' })}</p>
       </div>
-      <div className={styles.entryDayList}>
-        <div className={styles.entryListItem}>
-          <p className={styles.entryListItemTime}>
-            {Intl.DateTimeFormat('en', { hour: 'numeric', minute: 'numeric', hour12: true }).format(
-              entry.creationDateTime
-            )}
-          </p>
+      <div className={styles.entryListItemContainer}>
+        <Link href={`home/entries/${entry.slug}`}>
+          <div className={styles.entryListItem}>
+            <p className={styles.entryListItemTime}>
+              {Intl.DateTimeFormat('en', { hour: 'numeric', minute: 'numeric', hour12: true }).format(entry.created_at)}
+            </p>
 
-          <h2 className={styles.entryListItemTitle}>{entry.title}</h2>
-          <div className={styles.entryListItemBody}>
-            <p>{entry.body}</p>
+            <h2 className={styles.entryListItemTitle}>{entry.title}</h2>
+            <div className={styles.entryListItemBody}>
+              <p>{entry.body}</p>
+            </div>
           </div>
-        </div>
+        </Link>
       </div>
     </div>
   );
```

Comment:
There are conflicting philosophies around the usage of classes. I see here you are preferring to apply custom classes per element. 
While I'm not against this strategy, SCSS superimposes its own classes during compilation. As such, do you think there are any situations that we could lean on the power of SCSS rather than exhaustive classes?

Reply:
do you mean I could be utilizing more nesting instead? what do you mean by scss superimposing its own classes?
thx for the note though, I don't have a strong attachment to the way i've been writing classes, it's just an old habit

Reply:
I think I see what you mean with how you wrote layout.module.scss. I'll try a similar approach of targeting tags when I can, instead of defaulting to making a class.

Reply:
SCSS will compile and create custom classes for every relevant element. If you check your DOM you will see an that your class names will have an extra has associated to them. Even those that didnt get a className will still get a class after compilation.